Tools/Schema.NET.Tool/GeneratorModels/ICloneable{T}.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace Schema.NET.Tool.GeneratorModels
+namespace Schema.NET.Tool.GeneratorModels
 {
     public interface ICloneable<T>
     {
         T Clone();
     }
+
+    public interface ICloneable<TOut, TIn>
```

Comment:
I think you'll be getting a warning about putting two interfaces in one file.

Reply:
Part of me was thinking of dropping the interfaces entirely. We don't actually call through them directly as we have access to the concrete type.